cynthion/python/src/commands/util.py

```diff
@@ -16,22 +16,27 @@
 import textwrap
 import usb
 
+try:
+    from importlib_resources import files # <= 3.8
+except:
+    from importlib.resources import files # >= 3.9
+
 from cynthion                  import shared
 from fwup.dfu                  import DFUTarget
 from tqdm                      import tqdm
 
 
 SOC_FIRMWARE_FLASHADDR = 0x000b0000
 
+_MSG_SOURCE_INSTALL = """
+If you have installed the 'cynthion' Python package from source please
```

Comment:
This could be clearer. `make assets` doesn't solve the problem on its own. You have to `make assets` before installing cynthion.

Reply:
Ah yes, here too I failed to take into account that not everyone is going to `pip install -e` 🤦

Used better check to give a better message when it wa `pip install .`

Reply:
https://github.com/greatscottgadgets/cynthion/pull/166/commits/eeec9ae47a0eef628a65fe59df37ef5107da7adc